### DISCUSSION TITLE
v6.0-(1402-10-28)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#30] [Announcements]
برای گوشی جدید v8a 
برای قدیمی v7a 
و نسخه جنریک سازگار با همه گوشی ها

[کانال تلگرام برنامه](https://t.me/mahsa_net)

changes:
fragment & mux using core
filter resistance by multi-path config fetcher
telegram proxy menu
fast youtube config
progress bar and key debounce 
quick paste button
better ui design
fix bug of shared-pref async process

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v6.0-(1402-10-28)'>v6.0-(1402-10-28)</a>.</em>

**Comment by nimbold** [comment]
Thanks a lot for the reworked UI for fragment. It is easy to manage now and is working great 🍻

**Comment by mahxray** [comment]
salam dadash.
mishe file libv2ray.arr ke vase akharin version mahsaNG estefade shode ro behem bdin?

**Reply by GFW-knocker** [reply]
https://github.com/2dust/AndroidLibXrayLite

**Reply by mahxray** [reply]
> https://github.com/2dust/AndroidLibXrayLite

mamnoonam
